package.json

```diff
@@ -50,7 +52,7 @@
   "peerDependencies": {
     "@nestjs/common": "^9.0.0 || ^10.0.0",
     "@nestjs/core": "^9.0.0 || ^10.0.0",
-    "cache-manager": "<=5",
+    "cache-manager": ">=6",
```

Comment:
Projects use `cache-manager` through `@nestjs/cache-manager`, so why force them to explicitly specify the dependency?

`cache-manager@^6.1.0` could be made a direct dependency.

Reply:
It is a direct dependency already. Defining it as peer dependency as well allows consumers to provide their own version, e.g. let say 6.2.0 once it's released without a need to publish the lib again.

Reply:
I didn't notice that, but that makes even less sense. It should be either a direct dependency or a peer dependency, but not both at the same time.

Reply:
I personally don't agree, having both gives the best of both worlds. Folks that don't care about providing their own copy will get the direct dependency version, the rest will be able to shrink their dependency tree by sharing the version they already use.

Reply:
> shrink their dependency tree by sharing the version they already use (in case of major release).

Actually this makes sense for the current major version of `cache-manager` (but not for all future versions).

Seems `@nestjs/axios` also declares `axios` only as a peer dependency.

Reply:
Do you mean to specify the version directly? Maybe change it to `=6` ? I did get lazy and just wrote `>=6`, this package updates shouldn't be frequent, is specifying the version directly not a good idea either?